### PR TITLE
Add digital ocean post

### PR DESCRIPTION
**Pull request by calston**
Instructions for using DigitalOcean App Platform

### CODE REVIEW

File: _posts/2022-11-11-digital-ocean.markdown [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+![Complete signup](/images/digitalocean/2.png)
+
+## Create an application
+
+Click on `Deploy a web application` to get started. 
+
+![Deploy a web application](/images/digitalocean/create-app-1.png)
+
+Chose "Deploy your source code" to add an existing GitHub repository
```
Comment (tombruijn) [inline _posts/2022-11-11-digital-ocean.markdown:32]:
```suggestion
Chose "Deploy your Rails server" to add an existing GitHub repository
```

I don't find that exact phrase, but it should be for Rails apps?

File: _posts/2022-11-11-digital-ocean.markdown [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+## Configuration
+You can now head to the `Console` to access your application container and setup the database.
+
+Type `rails db:migrate` into the terminal and press Enter. You should see the database being setup with the Rails schema.
+
+![Migrate](/images/digitalocean/migrate.png)
+
+If all went well you should now be able to click on the `Live App` button which links to the live server.
+
+![Tada](/images/digitalocean/fin.png)
```
Comment (tombruijn) [inline _posts/2022-11-11-digital-ocean.markdown:79]:
```suggestion
![Tada](/images/digitalocean/fin.png)
## Conclusion

Your Rails app is now running in the cloud on DigitalOcean. You can push your changes to GitHub and they'll show up automatically the live URL after some time. Share the URL to show off your app to your friends!

Keep an eye out when your free credits run out and delete the app if you no longer need it.
```

**Review by tombruijn — APPROVED** [review]
Two small suggestions, otherwise looks good 👍

**Comment by tombruijn** [comment]
You should also link to this page from this list: https://github.com/railsgirls/railsgirls.github.io/blob/29f60dc570a504f64c512ff38e0ae0c307f69f90/_includes/other-guides.md#L9-L14

File: _posts/2022-11-11-digital-ocean.markdown (hunk 1 above)
Comment (leesheppard) [inline _posts/2022-11-11-digital-ocean.markdown:32]:
```suggestion
Choose "Deploy your source code" to add an existing GitHub repository
```

**Pull request merged**